src/js/components/Main/Home/Home.container.jsx

```diff
@@ -32,22 +63,33 @@ const HomeContainer = props => {
       [concerts]
   );
 
+  function onFocusOut() {
+    setTimeout(
+        () => {
+          setInputText("");
+          setHelpList([]);
+        },
+        100)
+  }
+
```

Comment:
this function uses for hiding tip block when typing in search field. Timeout setted because after clicking on tip it disappears and redirect don't go. How to write it in better way?